File: Java/app/src/main/java/dev/ogabek/java/activity/MainActivity.java
package dev.ogabek.java.activity;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.GridLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

import java.util.ArrayList;
import java.util.List;

import dev.ogabek.java.R;
import dev.ogabek.java.adapter.UserAdapter;
import dev.ogabek.java.model.User;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        initViews();

    }

    private void initViews() {
        RecyclerView recyclerView = findViewById(R.id.rv_main);

        recyclerView.setLayoutManager(new GridLayoutManager(this, 1));

        UserAdapter adapter = new UserAdapter(prepareUserList());
        recyclerView.setAdapter(adapter);
    }

    private List<User> prepareUserList() {
        List<User> userList = new ArrayList<>();
        userList.add(new User(R.drawable.ogabekdev, "Ogabek Matyakubov", "online"));
        userList.add(new User(R.drawable.bogibek, "Bogibek Aka", "online"));
        userList.add(new User(R.drawable.ogabekdev, "Ogabek Matyakubov", "online"));
        userList.add(new User(R.drawable.bogibek, "Bogibek Aka", "online"));
        userList.add(new User(R.drawable.jonibek, "Jonibek Holmonov", "online"));
        userList.add(new User(R.drawable.aziz, "Aziz"));
        userList.add(new User(R.drawable.elmurod, "Elmurod Aka"));
        userList.add(new User(R.drawable.jabbor, "Jabbor Aka"));
        userList.add(new User(R.drawable.shakhriyor, "Shakhriyor Usta"));
        userList.add(new User(R.drawable.yahyo, "Yahyo Mahmudiy"));
        userList.add(new User(R.drawable.aziz, "Aziz"));
        userList.add(new User(R.drawable.ogabekdev, "Ogabek Matyakubov"));
        userList.add(new User(R.drawable.bogibek, "Bogibek Aka"));
        userList.add(new User(R.drawable.jonibek, "Jonibek Holmonov"));
        userList.add(new User(R.drawable.elmurod, "Elmurod Aka"));
        userList.add(new User(R.drawable.jabbor, "Jabbor Aka"));
        userList.add(new User(R.drawable.shakhriyor, "Shakhriyor Usta"));
        userList.add(new User(R.drawable.yahyo, "Yahyo Mahmudiy"));
        userList.add(new User(R.drawable.aziz, "Aziz"));
        userList.add(new User(R.drawable.ogabekdev, "Ogabek Matyakubov"));
        userList.add(new User(R.drawable.bogibek, "Bogibek Aka"));
        userList.add(new User(R.drawable.jonibek, "Jonibek Holmonov"));
        userList.add(new User(R.drawable.elmurod, "Elmurod Aka"));
        userList.add(new User(R.drawable.jabbor, "Jabbor Aka"));
        userList.add(new User(R.drawable.shakhriyor, "Shakhriyor Usta"));
        userList.add(new User(R.drawable.yahyo, "Yahyo Mahmudiy"));
        userList.add(new User(R.drawable.ogabekdev, "Ogabek Matyakubov"));
        userList.add(new User(R.drawable.bogibek, "Bogibek Aka"));
        userList.add(new User(R.drawable.jonibek, "Jonibek Holmonov"));
        userList.add(new User(R.drawable.elmurod, "Elmurod Aka"));
        userList.add(new User(R.drawable.jabbor, "Jabbor Aka"));
        userList.add(new User(R.drawable.shakhriyor, "Shakhriyor Usta"));
        userList.add(new User(R.drawable.yahyo, "Yahyo Mahmudiy"));
        userList.add(new User(R.drawable.aziz, "Aziz"));
        userList.add(new User(R.drawable.ogabekdev, "Ogabek Matyakubov"));
        userList.add(new User(R.drawable.bogibek, "Bogibek Aka"));
        userList.add(new User(R.drawable.jonibek, "Jonibek Holmonov"));
        userList.add(new User(R.drawable.elmurod, "Elmurod Aka"));
        userList.add(new User(R.drawable.jabbor, "Jabbor Aka"));
        userList.add(new User(R.drawable.shakhriyor, "Shakhriyor Usta"));
        userList.add(new User(R.drawable.yahyo, "Yahyo Mahmudiy"));
        return userList;
    }
}

File: Kotlin/app/src/main/java/dev/ogabek/kotlin/activity/MainActivity.kt
package dev.ogabek.kotlin.activity

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.GridLayoutManager
import androidx.recyclerview.widget.RecyclerView
import dev.ogabek.kotlin.R
import dev.ogabek.kotlin.adapter.UserAdapter
import dev.ogabek.kotlin.model.User

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        initViews()

    }

    private fun initViews() {
        val recyclerView = findViewById<RecyclerView>(R.id.rv_main)
        recyclerView.layoutManager = GridLayoutManager(this, 1)
        val adapter = UserAdapter(prepareUserList())
        recyclerView.adapter = adapter
    }

    private fun prepareUserList(): List<User> {
        val userList: MutableList<User> = ArrayList<User>()
        userList.add(User(R.drawable.ogabekdev, "Ogabek Matyakubov", "online"))
        userList.add(User(R.drawable.bogibek, "Bogibek Aka", "online"))
        userList.add(User(R.drawable.ogabekdev, "Ogabek Matyakubov", "online"))
        userList.add(User(R.drawable.bogibek, "Bogibek Aka", "online"))
        userList.add(User(R.drawable.jonibek, "Jonibek Holmonov", "online"))
        userList.add(User(R.drawable.aziz, "Aziz"))
        userList.add(User(R.drawable.elmurod, "Elmurod Aka"))
        userList.add(User(R.drawable.jabbor, "Jabbor Aka"))
        userList.add(User(R.drawable.shakhriyor, "Shakhriyor Usta"))
        userList.add(User(R.drawable.yahyo, "Yahyo Mahmudiy"))
        userList.add(User(R.drawable.aziz, "Aziz"))
        userList.add(User(R.drawable.ogabekdev, "Ogabek Matyakubov"))
        userList.add(User(R.drawable.bogibek, "Bogibek Aka"))
        userList.add(User(R.drawable.jonibek, "Jonibek Holmonov"))
        userList.add(User(R.drawable.elmurod, "Elmurod Aka"))
        userList.add(User(R.drawable.jabbor, "Jabbor Aka"))
        userList.add(User(R.drawable.shakhriyor, "Shakhriyor Usta"))
        userList.add(User(R.drawable.yahyo, "Yahyo Mahmudiy"))
        userList.add(User(R.drawable.aziz, "Aziz"))
        userList.add(User(R.drawable.ogabekdev, "Ogabek Matyakubov"))
        userList.add(User(R.drawable.bogibek, "Bogibek Aka"))
        userList.add(User(R.drawable.jonibek, "Jonibek Holmonov"))
        userList.add(User(R.drawable.elmurod, "Elmurod Aka"))
        userList.add(User(R.drawable.jabbor, "Jabbor Aka"))
        userList.add(User(R.drawable.shakhriyor, "Shakhriyor Usta"))
        userList.add(User(R.drawable.yahyo, "Yahyo Mahmudiy"))
        userList.add(User(R.drawable.ogabekdev, "Ogabek Matyakubov"))
        userList.add(User(R.drawable.bogibek, "Bogibek Aka"))
        userList.add(User(R.drawable.jonibek, "Jonibek Holmonov"))
        userList.add(User(R.drawable.elmurod, "Elmurod Aka"))
        userList.add(User(R.drawable.jabbor, "Jabbor Aka"))
        userList.add(User(R.drawable.shakhriyor, "Shakhriyor Usta"))
        userList.add(User(R.drawable.yahyo, "Yahyo Mahmudiy"))
        userList.add(User(R.drawable.aziz, "Aziz"))
        userList.add(User(R.drawable.ogabekdev, "Ogabek Matyakubov"))
        userList.add(User(R.drawable.bogibek, "Bogibek Aka"))
        userList.add(User(R.drawable.jonibek, "Jonibek Holmonov"))
        userList.add(User(R.drawable.elmurod, "Elmurod Aka"))
        userList.add(User(R.drawable.jabbor, "Jabbor Aka"))
        userList.add(User(R.drawable.shakhriyor, "Shakhriyor Usta"))
        userList.add(User(R.drawable.yahyo, "Yahyo Mahmudiy"))
        return userList
    }

}